Lib/test/test_tools/test_pathfix.py

```diff
@@ -53,16 +53,48 @@ def test_pathfix_keeping_flags(self):
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python -R',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k', ]),
             '#! /usr/bin/python3 -R',
         )
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k', ]),
             '#! /usr/bin/python3',
         )
 
+    def test_pathfix_adding_flag(self):
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python',
+                ['-i', '/usr/bin/python3', '-a', 's', ]),
+            '#! /usr/bin/python3 -s',
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python -s',
+                ['-i', '/usr/bin/python3', '-a', 's', ]),
```

Comment:
I would prefer to use a different flag in the old shebang or in the -a argument. Ex: '#! /usr/bin/env python -s' and -a S.